### PR TITLE
Idea - Decentralization of Finance system

### PR DESCRIPTION
**Pull request by sahalhes**
## Description of the idea

Proposal for Decentralization of finance system which aligns with web3 and blockchain technology for future.

## Category

- Future Finance System

## Checklist

- [x] I have clearly mentioned the creator of the idea, whether it's mine or someone else's.
- [x] My entry adheres to the Docs style guide, ensuring consistency and readability.
- [x] My changes do not generate any new warnings or errors.
- [x] I have thoroughly checked my entries for misspellings and corrected them.
- [x] I have made necessary updates to the documentation if my changes impact it.
- [x] I have confirmed that my changes are not being pushed from my forked main branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making to keep the codebase organized and manageable.

### CODE REVIEW

File: ideaTemplate/ideas.md [hunk 1]
```diff
@@ -1 +1,48 @@
-Please consult the [idea template](ideaTemplate/README.md) prior to completing the details of your idea.
+## Author Details
+
```
Comment (ATREAY) [inline ideaTemplate/ideas.md:2]: just the title should be in block letters not the description of it.

File: ideaTemplate/ideas.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+- **Name of the author: E S SAHAL HUSSAIN**
+- **Twitter / LinkedIn : https://www.linkedin.com/in/sahalhes/**
+- **GitHub Link: https://github.com/sahalhes**
+- **Email: sahaldevelop@gmail.com**
+
+## Idea Details
+
```
Comment (ATREAY) [inline ideaTemplate/ideas.md:9]: The same applies to the idea's title.

File: ideaTemplate/ideas.md [hunk 3]
```diff
@@ -1 +1,48 @@
...
+No technical work is done yet by me, but I am sure this idea motivates me as well as the reader to learn more about technology and build.
+
+## Non-Technical:
+
+This proposal seeks to initiate a global conversation and collaborative effort to develop a decentralized banking system that addresses the shortcomings of centralization while promoting financial inclusion, security, and empowerment for individuals worldwide. By embracing the principles of decentralization and blockchain technology, we can create a financial system that is more resilient, accessible, and aligned with the needs and values of the 21st century (Web 3.0 and Blockchain Technology for instance). 
+
+Together, we can shape the future of finance for the better.
+
+Additiional Note from Author:
```
Comment (ATREAY) [inline ideaTemplate/ideas.md]: Just give it a quick spell check here

File: ideaTemplate/ideas.md [hunk 4]
```diff
@@ -1 +1,48 @@
...
+
+## Technical:
+
+No technical work is done yet by me, but I am sure this idea motivates me as well as the reader to learn more about technology and build.
+
+## Non-Technical:
+
+This proposal seeks to initiate a global conversation and collaborative effort to develop a decentralized banking system that addresses the shortcomings of centralization while promoting financial inclusion, security, and empowerment for individuals worldwide. By embracing the principles of decentralization and blockchain technology, we can create a financial system that is more resilient, accessible, and aligned with the needs and values of the 21st century (Web 3.0 and Blockchain Technology for instance). 
+
+Together, we can shape the future of finance for the better.
```
Comment (ATREAY) [inline ideaTemplate/ideas.md]: I think we can skip this sentence; it's a bit too vague.

**Review by ATREAY — CHANGES_REQUESTED** [review]
hey @sahalhes, I've requested for certain modifications. Kindly proceed with making those changes accordingly.

File: ideaTemplate/ideas.md [hunk 5]
```diff
@@ -1 +1,48 @@
...
+- **Name of the author: E S SAHAL HUSSAIN**
+- **Twitter / LinkedIn : https://www.linkedin.com/in/sahalhes/**
+- **GitHub Link: https://github.com/sahalhes**
+- **Email: sahaldevelop@gmail.com**
+
+## Idea Details
+
+- **Title: DECENTRALIZATION OF BANKS**
+- **Category:** 
+    - Future Finance System
```
Comment (TusharPuranik) [inline ideaTemplate/ideas.md]: Please modify the category here. It could be in the blockchain category.

**Review by TusharPuranik — CHANGES_REQUESTED** [review]
A lot of decentralized finance systems already exist, elaborate on how your idea differentiates from the rest.

**Comment by sahalhes** [comment]
@ATREAY Hey I have resolved all the issues told by both you and @TusharPuranik . Kindly check and tell if there are any more changes required.

**Review by ATREAY — APPROVED** [review]
Hey @sahalhes, it looks good to me! Now, let's wait and see what the other reviewer thinks.

**Review by TusharPuranik — APPROVED** [review]
Looks good.

**Comment by ATREAY** [comment]
Big thanks, @sahalhes! We really appreciate your contribution to openBookmarkz and for sharing your awesome idea with us. Your input means a lot to us and helps us grow. Keep those ideas coming!

**Pull request merged**